### DISCUSSION TITLE
Request to consider asdf-pgbadger plugin

### DISCUSSION THREAD

**Discussion by drgeb** [asdf-community/.github#35] [General]
Hi,
I wrote a plugin for PgBadger at https://github.com/drgeb/asdf-pgbadger

I would like to move the new plugin here.

Thanks

**Comment by smorimoto** [comment]
Consult this discussion post: https://github.com/orgs/asdf-community/discussions/57 This is the smoothest way to go. Of course I can do all the work for you, but it's done on a best effort basis. (I've been busy with work lately...)
